src/js/components/Data/Data.js

```diff
@@ -51,6 +53,41 @@ export const Data = ({
     return filter(dataProp, view, properties);
   }, [dataProp, filteredTotal, onView, properties, total, view]);
 
+  const announce = useContext(AnnounceContext);
+  const { format } = useContext(MessageContext);
+  useEffect(() => {
```

Comment:
Add some sort of comment explaining the intent behind this useEffect

```suggestion
  // Announce to screen readers when search or filters are applied and affect the underlying result set
  useEffect(() => {
```